src/Compiler/CompilerOptions.cpp

```diff
@@ -23,59 +23,66 @@
-bool printIR;                                  // onnx-mlir only
-bool preserveBitcode;                          // onnx-mlir only
-bool preserveLLVMIR;                           // onnx-mlir only
-bool preserveMLIR;                             // onnx-mlir only
-bool useOnnxModelTypes;                        // onnx-mlir only
-int repeatOnnxTransform;                       // onnx-mlir only
-std::string shapeInformation;                  // onnx-mlir only
-ModelSize modelSize;                           // onnx-mlir only
-bool storeConstantsToFile;                     // onnx-mlir only
-float constantsToFileTotalThreshold;           // onnx-mlir only
-float constantsToFileSingleThreshold;          // onnx-mlir only
-bool VerboseOutput;                            // onnx-mlir only
-std::vector<std::string> Xopt;                 // onnx-mlir only
-std::vector<std::string> Xllc;                 // onnx-mlir only
-std::string mllvm;                             // onnx-mlir only
-std::string instrumentOps;                     // onnx-mlir only
-unsigned instrumentControlBits;                // onnx-mlir only
-bool instrumentONNXSignature;                  // onnx-mlir only
-std::string ONNXOpStats;                       // onnx-mlir only
-bool enableMemoryBundling;                     // onnx-mlir only
-int onnxOpTransformThreshold;                  // onnx-mlir only
-bool onnxOpTransformReport;                    // onnx-mlir only
-bool enableParallel;                           // onnx-mlir only
-bool disableSimdOption;                        // onnx-mlir only
-bool enableSimdDataLayout;                     // onnx-mlir only
-bool verifyInputTensors;                       // onnx-mlir only
-bool allowSorting;                             // onnx-mlir only
-std::string reportHeapBefore;                  // onnx-mlir only
-std::string reportHeapAfter;                   // onnx-mlir only
-std::string modelTag;                          // onnx-mlir only
-bool enableConvOptPass;                        // onnx-mlir only
-std::vector<std::string> extraLibPaths;        // onnx-mlir only
-std::vector<std::string> extraLibs;            // onnx-mlir only
-ProfileIRs profileIR;                          // onnx-mlir only
-OnnxOpReport onnxOpReport;                     // onnx-mlir only
-bool split_input_file;                         // onnx-mlir-opt only
-bool verify_diagnostics;                       // onnx-mlir-opt only
-bool verify_passes;                            // onnx-mlir-opt only
-bool allowUnregisteredDialects;                // onnx-mlir-opt only
+
+// Options known to both onnx-mlir and onnx-mlir-opt:
+std::string inputFilename;
+std::string outputBaseName;
+std::vector<accel::Accelerator::Kind> maccel;
+OptLevel OptimizationLevel;
+std::string mtriple;
+std::string mcpu;
+std::string march;
+InstrumentStages instrumentStage;
+int onnxConstPropExpansionBound;
+std::vector<std::string> onnxConstPropDisablePatterns;
```

Comment:
@gongsu832 fyi, I refactored the comments because the old style got in the way of this long line

Reply:
@sorenlassen if you add the line
```
std::vector<std::string> onnxConstPropDisablePatterns; // common for both
```
then run `clang-format-9`, it will indent all the lines properly. I prefer the old style because when the list don't fit in my window I don't have to scroll back to remember which one of the 3 sections I'm currently in.

Reply:
ok, will do

just beware that clang-format will do something funny to avoid exceeding the line length limit (and it may break down further in the future if people add more long lines)

Reply:
done

Reply:
> just beware that clang-format will do something funny to avoid exceeding the line length limit (and it may break down further in the future if people add more long lines)

When that happens you know your option name is too long and needs to be shortened. :-)